galaxykit/users.py

```diff
@@ -69,7 +69,6 @@ def create_user(
 def update_user(client, user):
     return client.put(f"_ui/v1/users/{user['id']}/", user)
 
-
```

Comment:
looks like 2 empty lines between methods is a convention used in this project

Reply:
Yes - we use the `black` code formatter, which defaults to 2 lines between methods. I'm making a note to ensure I put it in the contributors guide I'm going to write up this afternoon.